### PR TITLE
Improve and fix docs which had incorrect stub info

### PR DESCRIPTION
**Pull request by Girgias**
Not sure what the wording usually is for the changelog and the error sections, so please advise.

### CODE REVIEW

File: reference/exec/functions/exec.xml [hunk 1]
```diff
@@ -79,6 +79,42 @@
...
+     <row>
+      <entry>&Version;</entry>
+      <entry>&Description;</entry>
+     </row>
+    </thead>
+    <tbody>
+     <row>
+      <entry>8.0.0</entry>
+      <entry>
+       If <parameter>command</parameter> is empty or contain null bytes
```
Comment (kocsismate) [inline reference/exec/functions/exec.xml:108]:
```suggestion
       If <parameter>command</parameter> is empty or contains null bytes
```

Comment (kocsismate) [reply reference/exec/functions/exec.xml:108]: I'm not sure about the rest of the sentence, but would this make more sense?
```
If <parameter>command</parameter> is empty or contains null bytes,
it throws a <classname>ValueError</classname> now.
```

Comment (cmb69) [reply reference/exec/functions/exec.xml:108]: I think using `<function>exec</function>` (or maybe "this function") instead of "it" would be more appropriate for technical documentation.

File: reference/gmp/functions/gmp-random-seed.xml [hunk 2]
```diff
@@ -37,18 +37,42 @@
...
+   &return.void;
   </para>
  </refsect1>
 
  <refsect1 role="errors"><!-- {{{ -->
   &reftitle.errors;
   <para>
-   Issues an <constant>E_WARNING</constant> and returns &false; if
-   <parameter>seed</parameter> is not valid.
+   Throws a <classname>ValueError</classname> is <parameter>seed</parameter>
```
Comment (kocsismate) [inline reference/gmp/functions/gmp-random-seed.xml:47]:
```suggestion
   Throws a <classname>ValueError</classname> if <parameter>seed</parameter>
```

File: reference/exec/functions/passthru.xml [hunk 3]
```diff
@@ -61,6 +61,42 @@
...
 
+ <refsect1 role="errors">
+  &reftitle.errors;
+  <para>
+   Will emit an <constant>E_WARNING</constant> if <function>passthru</function>
+   is unable to fork the <parameter>command</parameter>.
+  </para>
+  <para>
+   Throws a <classname>ValueError</classname> if <parameter>command</parameter>
+   is empty or contain null bytes.
```
Comment (kocsismate) [inline reference/exec/functions/passthru.xml:72]: contain -> contains (there are a bunch of similar issues above)

File: reference/exec/functions/exec.xml [hunk 4]
```diff
@@ -79,6 +79,42 @@
   </para>
  </refsect1>
 
+ <refsect1 role="errors">
+  &reftitle.errors;
+  <para>
+   Emits an <constant>E_WARNING</constant> if <function>exec</function>
+   is unable to fork the <parameter>command</parameter>.
```
Comment (cmb69) [inline reference/exec/functions/exec.xml:86]: It might be better not to use "fork" here. On Windows, processes are never forked (that's just not supported). If we want to be precise, we should clarify that in the description section.
```suggestion
   is unable to execute the <parameter>command</parameter>.
```

File: reference/exec/functions/passthru.xml [hunk 5]
```diff
@@ -61,6 +61,42 @@
   </para>
  </refsect1>
 
+ <refsect1 role="errors">
+  &reftitle.errors;
+  <para>
+   Will emit an <constant>E_WARNING</constant> if <function>passthru</function>
+   is unable to fork the <parameter>command</parameter>.
```
Comment (cmb69) [inline reference/exec/functions/passthru.xml:68]: Same as above
```suggestion
   is unable to execute the <parameter>command</parameter>.
```

File: reference/funchand/functions/register-shutdown-function.xml [hunk 6]
```diff
@@ -9,7 +9,7 @@
  <refsect1 role="description">
   &reftitle.description;
   <methodsynopsis>
-   <type class="union"><type>bool</type><type>null</type></type><methodname>register_shutdown_function</methodname>
+   <type>void</type><methodname>register_shutdown_function</methodname>
```
Comment (cmb69) [inline reference/funchand/functions/register-shutdown-function.xml:12]: This is only [correct as of PHP 8.2.0](https://heap.space/search?project=PHP-8.1&project=php-src&full=&defs=register_shutdown_function&refs=&path=&hist=&type=). I suggest to move this change to another PR (with the proper milestone), and to add a changelog entry there.

**Review by cmb69 — COMMENTED** [review]
Thank you!

File: reference/funchand/functions/register-shutdown-function.xml (hunk 6 above)
Comment (Girgias) [reply reference/funchand/functions/register-shutdown-function.xml:12]: Did you mean 8.1? (or did the patch get backported in between) But will add a changelog

Comment (cmb69) [reply reference/funchand/functions/register-shutdown-function.xml:12]: Hmm, I likely was confused about the versions. So just a changelog entry.

Comment (Girgias) [reply reference/funchand/functions/register-shutdown-function.xml:12]: So I just got back to this, and as far as I can see, false was only returned when the callable was invalid, which is since 8.0 a proper ZPP failure. So don't think we need a changelog?

Comment (cmb69) [reply reference/funchand/functions/register-shutdown-function.xml:12]: Indeed, you are right. The [PHP 8.0 and 8.1 stubs are wrong](https://heap.space/search?project=PHP-8.0&project=PHP-8.1&full=&defs=register_shutdown_function&refs=&path=&hist=&type=), though.

**Review by cmb69 — APPROVED** [review]
